posts/2020-08-28-graphql-apis-with-open-liberty-20.0.0.9.adoc

```diff
@@ -0,0 +1,145 @@
+
+
+=== Change your SSL config without restarting your server
+
+If you use the JAX-RS Client in your application to access SSL-secured RESTful services, then you will likely have a key store and/or trust store configured. These SSL settings enable you to ensure you are communicating with the endpoint you expect - and that nobody else is listening in!  These settings also enable SSL-based client authentication.
+Whatever you use these SSL settings for, if you need to change them, then that meant that you would need to restart your server so that the clients could pick up the new settings.  Well, that is a thing of the past! Starting in 20.0.0.9, JAX-RS clients will now be able to dynamically adjust to changes in the SSL configuration.  This should rapidly improve app development and deployment!
+
+* Resolved in issue link:https://github.com/OpenLiberty/open-liberty/issues/13027[#13027]
+
+
+=== Occasional error during Arquillian testing
+
+During Arquillian testing there was a bug that would throw an occasional `ArrayIndexOutOfBoundsException` in `JaspiServiceImpl.getDescription`. This problem was caused by registering the internal `AuthConfigProvider` for JSR 375. There are two threads registering two instances of the provider. Additional synchronization has been added to that a single internal provider is registered.
+
+* Resolved in issue link:https://github.com/OpenLiberty/open-liberty/issues/11504[#11504]
+
+
+=== Bug causing include tag not to be parsed with featureUtility
+
+This bug was caused where the include tag in `server.xml` was being ignored by featureUtility. The include element can be used in a `server.xml` in order to consolidate a configuration from a separate `server.xml` file. The bug was fixed by handling the parsing with a more appropriate XML method.
+
+* Resolved in issue link:https://github.com/OpenLiberty/open-liberty/issues/13138[#13138]
+
+
+=== MicroProfile Graphql Exception allowlist now up and running 
+
+As an open source product, we love hearing feedback from Open Liberty users.  We especially love comments like this: "Hello, I am using microprofile-graphql on openliberty and everything goes well…" and even the rest of that sentence (taken from Open Liberty https://github.com/OpenLiberty/open-liberty/issues/13036[#13036]): "except for the exception whitelisting mechanism via microprofile config described here."
+Our MicroProfile GraphQL feature has only been generally available for a few months, so it's great to see that users are using it - and it's exciting to see that they are already exploring the "dark corners" (exception handling, etc.).  While we hate to find out that we let a bug slip through the cracks, we're always eager to fix them when they do. 
```

Comment:
```suggestion

Our MicroProfile GraphQL feature has only been generally available for a few months, so it's great to see that users are using it - and it's exciting to see that they are already exploring the "dark corners" (exception handling, etc.).  While we hate to find out that we let a bug slip through the cracks, we're always eager to fix them when they do. 

```

Suggested line breaks to separate paragraphs.

Reply:
Done